proxy/api/src/identity.rs

```diff
@@ -92,6 +105,8 @@ pub struct Metadata {
     pub handle: String,
     /// The user's Ethereum address.
     pub ethereum: Option<Ethereum>,
+    /// The user's PeerIds.
+    pub peer_ids: Option<Vec<PeerId>>,
```

Comment:
I tried it without wrapping it in an `Option` initially. But then the app wouldn't start because the session endpoint expected `peer_ids` to be set in the metadata.